TipCatDotNet.Api/Models/HospitalityFacilities/PaymentRequest.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace TipCatDotNet.Api.Models.HospitalityFacilities
+{
+    public class PaymentRequest
+    {
+        public PaymentRequest(int receiverId, int tipeSize, string cardNumber, DateTime cardCreationDate, DateTime cardExpirationDate, 
+            string cardOwner, int cvv)
+        {
+            ReceiverId = receiverId;
+            TipSize = tipeSize;
+            CardNumber = cardNumber;
+            CardCreationDate = cardCreationDate;
+            CardExpirationDate = cardExpirationDate;
+            CardOwner = cardOwner;
+        }
+
+        [Required]
+        public int ReceiverId { get; }
+        [Required]
+        public int TipSize { get; }
+        [Required]
+        public string CardNumber { get; }
```

Comment:
As for card details, we unable to collect them, so this part depends totally on Stripe API. Let remove them for now.

Reply:
I'll describe the payment process later.